cmd/sonobuoy/app/gen_plugin.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2018 Heptio Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+import (
+	"os"
+
+	ops "github.com/heptio/sonobuoy/cmd/sonobuoy/app/operations"
+	"github.com/heptio/sonobuoy/pkg/errlog"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var genPluginOpts ops.GenPluginConfig
+
+func init() {
+	cmd := &cobra.Command{
+		Use:   "plugin",
```

Comment:
I debate if this should be a subcommand of gen and if so where should it live.  1st classing it to the root seems weird to me.

Reply:
I agree it's weird, but I couldn't think of any better verbs to use.

Reply:
I'm ok with merging for now and revisiting the shuffle later.